### ISSUE TITLE
Can you make a view that doesn't have any images?

### ISSUE THREAD

**Issue by iconoclasthero** [tamland/airsonic-refix#134]
I wish I could have something that would have a listing of the elements of the library in TEXT ONLY or some way to turn off album covers...  I have no idea why I need a bunch of empty placeholders to show that I have a bunch of bootlegs, the names of which are cropped.  Honestly, this has been the source of a remarkable degree of frustration since I cannot stand the images ...and well, I best stop there.

![Screenshot from 2024-03-26 23-20-37](https://github.com/tamland/airsonic-refix/assets/535962/10b824c5-4155-40d1-bf27-923529defcfc)

**Comment by iconoclasthero** [comment]
Also,what is that 36 albums listed **on an entire screen!**  I assume it would be possible to waste more screen space with fewer albums, but it would be difficult.  Sorry, but lots of people have thousands of albums—while I can't speak for any of them—I don't have the time left in my life to go through it 36 albums at a time.

**Comment by iconoclasthero** [comment]
Just want a simple list.  Why is that so hard?

![Screenshot from 2024-03-26 23-27-24](https://github.com/tamland/airsonic-refix/assets/535962/b6cd2974-9a41-4507-8fa3-7d3668399bf4)

**Comment by sentriz** [comment]
why do you have such a bad attitude towards a free software project? it comes across as quite ungrateful. if you don't like something, fix it yourself

**Comment by iconoclasthero** [comment]
Yeah, it comes off as a terrible attitude, I am sorry about that.  It just breaks my mind every time I see this, it's so frustrating that I lose all perspective of the important things in life and end up pissing off the people that could and probably would help if I could manage to phrase things in a more palatable way.

**Comment by iconoclasthero** [comment]
![Screenshot from 2024-03-29 10-09-18](https://github.com/tamland/airsonic-refix/assets/535962/656ba41b-0cc9-451e-a390-fb762a2a6c1f)
Before you close this because i'm an asshole and whatnot, I just want to point out how not useful this type of display is in this instance,  First of all, I can't read the names of the bootlegs.  Likely nothing can be done about their names since that's how they're entered into MBDB and that's how they are and will continue to be organized on disc.  Further, they're actual bootlegs, like etree bootlegs, not the quasi-commercial ones that have covers.  Sure some might have images on archive.org, but a lot of them come from shnflac.net / lossless 'legs and they don't come with any .jpgs...and they never will, i mean it is a concert bootleg, not a video, what do i need a picture for...here i go again getting pissed off.

tl;dr, i feel _viscerally_ that the display is worthless;  I can't find anything and _that_ provokes very strong feelings.  i don't see any context in which this type of display is efficient and useful and it seems very much to me like people are doing it because there's some aesthetic ideal for the design of these clients that everyone's trying to meet instead of a bottom-up form-follows-function approach that looks at how someone actually finds and queues up music efficiently.

**Comment by sentriz** [comment]
maybe you would prefer the "Files" browser instead?

**Comment by iconoclasthero** [comment]
> maybe you would prefer the "Files" browser instead?

since gonic is the back end, that is a possibility, but as discussed on #gonic irc last/this week the request is still for an album view with no cover art displayed.

**Comment by tamland** [comment]
#26
